src/com/mediaengine/rpc/rpc_cdm_mediaengine_handler.h

```diff
@@ -33,20 +38,29 @@ namespace media {
  */
 class RpcCdmMediaengineHandler : public OpenCdmMediaengineCom {
  public:
+#ifdef WPE
+  RpcCdmMediaengineHandler(char *session_id_val, uint32_t session_id_len,
+                           uint8_t *auth_data_val, uint32_t auth_data_len);
+#else	//chrome
   static RpcCdmMediaengineHandler& getInstance();
 
   bool CreateMediaEngineSession(char *session_id_val, uint32_t session_id_len,
                            uint8_t *auth_data_val, uint32_t auth_data_len);
```

Comment:
See comment above about 'undoing' the work in commit 

Author: Zoltan Kuscsik <zoltan.kuscsik@linaro.org>  2015-10-16 14:54:18
Committer: Zoltan Kuscsik <zoltan.kuscsik@linaro.org>  2015-10-16 14:54:18
Parent: 02675213391e70305699ace63ebdc0e49fb46047 (Chromium: no symbolic links for OCDM folder)
Branches: chromium-53.0.2785.143, remotes/origin/chromium-53.0.2785.143, remotes/origin/master
Follows: 
Precedes: 

    Make RPC handler a singleton
    
    This is the first step to towards supporting
    multiple CDM session. The RPC handler needs
    to be a singleton.